_includes/tutorial_header.html

```diff
@@ -8,7 +8,9 @@
     <title>B'more on Rails Workshop for Women</title>
     <link href="{{ site.baseurl }}/css/syntax/monokai.css" type="text/css" rel="stylesheet"/>
     <link href="{{ site.baseurl }}/css/base.css" type="text/css" rel="stylesheet"/>
+    <link href="{{ site.baseurl }}/assets/js/featherlight-1.7.0/release/featherlight.min.css" type="text/css" rel="stylesheet" />
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
+    <script src="{{ site.baseurl }}/assets/js/featherlight-1.7.0/release/featherlight.min.js" type="text/javascript" charset="utf-8"></script>
```

Comment:
Hi! I'm thinking - since featherlight only requires these two files (featherlight.min.js and featherlight.min.css) and jquery ... could we just remove everything other than those?